ARCHITECTURE.md

```diff
@@ -11,18 +11,23 @@ our [API documentation][hoomd_documentation] on readthedocs.
 
 [hoomd_documentation]: https://hoomd-blue.readthedocs.io/en/latest/
 
+## Hardware and software support
+
+Each minor and major release of HOOMD-blue at a minimum supports:
+
+* x86_64 CPUs released in the four prior years.
+* NVIDIA GPUs released in the four prior years.
```

Comment:
Can we explicitly state what compute capabilities those are?

Reply:
We could, but then we would have to update this document regularly.

Reply:
Perhaps we could just state the minimum compute capability we are going to support? Then it wouldn't need to be updated nearly as much.

Reply:
Documenting changes in the minimum required compute capability would be more visible in the change log, along with the other mentions of support removed or added for compilers and libraries.

Reply:
I retroactively added this to the 3.0.0-beta.1 change log.